Python/pywarpx/picmi.py

```diff
@@ -272,7 +272,9 @@ def init(self, kw):
 
 
 class BinomialSmoother(picmistandard.PICMI_BinomialSmoother):
-    pass
+    def initialize_inputs(self):
+        pywarpx.warpx.use_filter = 1
+        pywarpx.warpx.filter_npass_each_dir = self.n_pass
```

Comment:
This may not work with the standard PICMI example:
[Here](https://github.com/picmi-standard/picmi/blob/master/Examples/laser_acceleration/laser_acceleration_PICMI.py#L154), we do `BinomialSmoother(n_pass = 1)`, so `self.n_pass` is a single integer. However, `WarpX` expects a vector (2 or 3 integers depending on the dimension).

@dpgrote Could you edit the PR accordingly?

Reply:
I can make the change.

Should the PICMI standard be updated, since it specifies that n_pass is a vector?

Reply:
Also, in WarpX, if n_pass is not specified, should it be given a default value, presumably 1?

Reply:
Yes, I think that the standard should be updated and require (for the moment) that the number of passes is a single integer.

(We should discuss how to handle cases where the number of passes can be different in different directions, and how to have a single synthax that would work in a single script that can do 2D, RZ or 3D geometry.)

Reply:
It would work if n_pass is either an integer or a vector. A vector would be geometry specific.